### ISSUE TITLE
RFE: please provide a Release page with links to all release notes

### ISSUE THREAD

**Issue by MagicFab** [nuxsmin/sysPass#925]
I'd like to propose a single web page including all links to release notes or an easy link to stable-release notes. I know they are tagged in GitHub, but having a single URL would reduce time to check and most importantly share this information. It's also not easy to discover this information for new users.

This links looks like it will always brings up the latest release notes for non-beta:
* https://github.com/nuxsmin/sysPass/releases/latest

This link looks like it will include all releases (beta + non-beta):
* https://github.com/nuxsmin/sysPass/tags

This link (which I only found when being logged in to a Syspass instance) directs to a page with Notices:
* https://github.com/nuxsmin/sysPass/labels/Notices

Perhaps a "StableRelease" label could be used to link directly as I mention above.

Then such a link could be added to a Releases page. Another link to "News" could link to the "notices" label (literal translation for "Noticias"?).

**Comment by nuxsmin** [comment]
@MagicFab you're right, a more helpful information about releases would be nice. I'll write up some info about major changes and features, although I take care to give some feedback on every commit.

Notices would be translated to 'avisos'.

Thanks for the feedback!
